trlx/trlx.py

```diff
@@ -1,111 +1,117 @@
 import os
+import warnings
 from typing import Callable, Dict, Iterable, List, Optional, Tuple
 
 from trlx.data.configs import TRLConfig
 from trlx.utils import set_seed
 from trlx.utils.loading import get_orchestrator, get_pipeline, get_trainer
 
 
-def train(
+def train(  # noqa: C901
+    config: Optional[TRLConfig] = None,
     model_path: Optional[str] = None,
```

Comment:
Maybe model path should still be the first arg? It's also whats still in the readme.

Reply:
yeah, reverted that bit